lib/Common/Memory/HeapInfo.h

```diff
@@ -496,7 +496,16 @@ HeapInfo::RealAlloc(Recycler * recycler, size_t sizeCat, size_t size)
 {
     Assert(HeapInfo::IsAlignedSmallObjectSize(sizeCat));
     auto& bucket = this->GetBucket<(ObjectInfoBits)(attributes & GetBlockTypeBitMask)>(sizeCat);
-    return bucket.template RealAlloc<attributes, nothrow>(recycler, sizeCat, size);
+    if (CONFIG_FLAG(ForceSoftwareWriteBarrier)
```

Comment:
my plan was removing this flag after the feature is done. I think yes there's perf regression but not sure how much will it be.
let put it under GLOBAL_ENABLE_WRITE_BARRIER, so only special build will have this on windows

---
In reply to: [88134740](https://github.com/Microsoft/ChakraCore/pull/1968#discussion_r88134740) [](ancestors = 88134740)